### ISSUE TITLE
b-nav-item-dropdown disabled no longer working

### ISSUE THREAD

**Issue by ShaynCornwell** [bootstrap-vue/bootstrap-vue#3264]
### Describe the bug

I updated from version 2.0.0-rc.14 to 2.0.0-rc.19 and since then setting the <b-nav-item-dropdown> disabled property no longer disables the dropdown item. I am using bootstrap-vue in a typescript component. 

it is possible this has nothing to do with the bootstrap-vue version. I switched back to RC14 and the links still look enabled, even with disabled property. 

**UPDATE**: So the Link is disabled, however, it seems webpack is not adding the "disabled" class to the link and it still looks active. Clicking on it just links to a standard # anchor link.

**WORKAROUND**: The problem is the Class on the <a> html element generated does not include the "disabled" class evenw ehn disabled is set. the work around is below
```
 <b-nav-item-dropdown  disabled>
                       toggle-class="disabled"
</b-nav-item-dropdown>
```

**Original Code:**
```

 <b-nav-item-dropdown  disabled>
</b-nav-item-dropdown>
```

Full set of code below:
```
<b-navbar-nav>
    <b-nav-item-dropdown
        class="m-md-2 main-menu"
         text="Main" >
        <b-dropdown-header class="submenu-header">Reports</b-dropdown-header>
            <b-dropdown-divider></b-dropdown-divider>
             <b-dropdown-item v-bind:to="{ name: 'create' }">Create</b-dropdown-item>
        </b-nav-item-dropdown>
         <b-nav-item-dropdown  class="m-md-2 main-menu"
                                         text="Codes Admin"
                                         disabled>
          </b-nav-item-dropdown>
</b-navbar-nav>
```

**Libraries:**
    "axios": "^0.18.0",
    "bootstrap": "^4.3.1",
    "bootstrap-vue": "^2.0.0-rc.19",
    "vue": "^2.6.8",
    "vue-class-component": "^7.1.0",
    "vue-property-decorator": "^7.3.0",
    "vue-router": "^3.0.6",
    "vue-session": "^1.0.0",
    "vuex": "^3.1.1",
    "vuex-class": "^0.3.1",
    "vuex-module-decorators": "^0.8.0",
    "vue-template-compiler": "^2.6.8",

**Comment by tmorehouse** [comment]
Disabled is working for me in the `dev` branch, but clicking on it causes a screen scroll to top (which it shouldn't).

We should be able to have this fixed in 2.0.0-rc.20.

**Comment by ShaynCornwell** [comment]
> Disabled is working for me in the `dev` branch, but clicking on it causes a screen scroll to top (which it shouldn't).
> 
> We should be able to have this fixed in 2.0.0-rc.20.

Does your disabled link  look disabled? mine look like active links with ::hover CSS. But I concur, my links are disabled, and scroll tot he top as well, however they look active.

**Comment by tmorehouse** [comment]
I think I know what the problem is.... and should be easy to fix.  Hoping to have 2.0.0-rc.20 out tomorrow.
